presto-docs/src/main/sphinx/release/release-0.262.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.262
+=============
+
+**Highlights**
+==============
+* Fix the issue of queries getting stuck in the queued state in disaggregated coordinator cluster.
```

Comment:
My understanding that since disagg coordinator is not announced - we don't need to announce fixes as well. @tdcmeehan to confirm

Reply:
Yes, please remove for now.